starter-code/package.json

```diff
@@ -8,14 +8,22 @@
     "dev-windows": "nodemon ./bin/www"
```

Comment:
Just a suggestion, you can add "seed": "node seeds/seed.js" so you have a command to seed the database